webapp/components/topbar/menuConfiguration.js

```diff
@@ -335,15 +328,15 @@ export const getModelMenu = (props) => (
       list: [
         {
           label: 'Automatic creation',
-          icon: props.automaticInstantiation ? checkedIcon : 'fa',
+          icon: !props.automaticInstantiation ? checkedIcon : 'fa',
```

Comment:
we want to change the default value of `automaticInstantation`, this is set in `general.js` and passed to the component in `components/index.js`.

With the current change you're inverting the logic for the two items (Automatic Creation and Manual Creation). We haven't removed this menu yet, so we want to keep the existing logic. By inverting the prop you would get Automatic Simulation when switching to Manual Creation and vice versa